code/game/machinery/computer/camera_advanced.dm

```diff
@@ -5,7 +5,7 @@
 	icon_keyboard = "security_key"
 	var/list/z_lock = list() // Lock use to these z levels
 	var/lock_override = NONE
-	var/mob/camera/aiEye/remote/eyeobj
+	var/mob/camera/ai_eye/remote/eyeobj
```

Comment:
ai_eye change is unrelated to this PR and should not be included with this PR. It will cause annoyances when porting.

Reply:
Porting from where? TG? They have this name changed too.

Reply:
well ok then